hashring/hashring_test.go

```diff
@@ -310,7 +310,8 @@ func TestLookupNLoopAround(t *testing.T) {
 	ring.AddMembers(members...)
 
 	unique := make(map[valuetype]struct{})
-	ring.tree.LookupNUniqueAt(1, replicaPoint{hash: 0}, unique)
+	orderedUnique := make([]valuetype, 0, 1)
+	ring.tree.LookupNUniqueAt(1, replicaPoint{hash: 0}, unique, &orderedUnique)
```

Comment:
Any suggestions on how to test this given the way the tests are written so far? Naively, I could just iterate a few times and confirm that the returned results for multiple interations of the same call are stable (i.e. same order), but that still wouldn't test that the order is what we expect.